Shopstagram - rozwiązanie/css/index.css

```diff
@@ -515,7 +514,7 @@ input:focus {
 .post {
   margin-bottom: 20px;
   background-color: #ffffff;
-  border: 1px solid #535233;
+  border: 1px solid #969696;
```

Comment:
Dlaczego tutaj nie widać zmiennych przypisanych do kolorów, tylko zamiast tego widać kod hex?
U mnie widać w kodzie jest:
.post {
    margin-bottom: 20px;
    background-color: $colorWhite;
    border: 1px solid $secondaryDarker;
    border-radius: 7px;

Reply:
bo to jest kod CSS już, a nie SCSS, tutaj wartość zmiennej jest już wyciągana, czyli nie widzisz zmiennej ale to co ona przechowywała